CDTDatastore/touchdb/TD_Database+Insertion.m

```diff
@@ -440,7 +445,8 @@ - (TD_Revision*)putRevision:(TD_Revision*)rev
                                     parentSequence:parentSequence
                                            current:YES
                                               JSON:json
-                                          database:db];
+                                          database:db
+                                             error:NULL];
```

Comment:
Wouldn't we be better making use of the error to differentiate between db full errors and general db errors as done elsewhere?